### PR TITLE
fix generate signatue

### PR DESCRIPTION
**Pull request by mxgrim**
If send data type "e-mail" then binance return: [Signature for this request is not valid.]

When using the Binance Corporate API

If need to add functions for working with the Binance Corporate API, just tell me how to name the functions correctly =)

Now:
sub_accounts() - [/sapi/v1/sub-account/list]
sub_account_assets() - [...]
sub_account_deposit_address() - [...]
sub_account_deposits() - [...]
sub_account_transfer_universal() - [...]

### CODE REVIEW

**Comment by mxgrim** [comment]
how long to wait for approval?

**Review by dmzoneill — APPROVED** [review]

**Pull request merged**
